src/borg/archiver.py

```diff
@@ -2699,6 +2702,8 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                               help='do not store ctime into archive')
         fs_group.add_argument('--ignore-inode', dest='ignore_inode', action='store_true',
                               help='ignore inode data in the file metadata cache used to detect unchanged files.')
+        fs_group.add_argument('--use-mtime', dest='use_mtime', action='store_true',
```

Comment:
use `--cache-mtime` and args.cache_mtime to make it more clear that this is cache related?

we also have `--noctime` and `--noatime`, but they are completely unrelated.